.claude/skills/ghflow-project-setup/tests/test_scripts.sh

```
@@ -0,0 +1,64 @@
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+REPO_ROOT="$(cd "$SCRIPT_DIR/../../../.." && pwd)"
+source "$REPO_ROOT/.claude/skills/shared/test_utils.sh"
+
+echo "==================================================================="
+echo "Testing ghflow-project-setup Scripts"
+echo "==================================================================="
+echo "Repository: $REPO_FULL"
+echo ""
+
+ORIGINAL_BRANCH=$(git branch --show-current)
+trap "git checkout $ORIGINAL_BRANCH 2>/dev/null || true" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)